### PR TITLE
Pre-release Q1 2018

### PR DESCRIPTION
**Pull request by alexfjw**
### Main Changes

The left halves of the images represent the new version, while the right represent the old.
Click the images to see the screenshots in full resolution.

#### Positions Lab
- One line added to mention that a single image will be aligned to the slide (above the diagram)
screen shot 2018-03-01 at 7 57 33 am

#### Shapes Lab
- New instructions for adding a shape, through the pane's "ADD" button (just like in SyncLab)
screen shot 2018-03-01 at 7 59 15 am

#### Shortcuts Lab
- New button, Paste To Fit Slide (full text below, written to resemble Paste To Fill Slide section found above this text. No comparison here as the section is new)

screen shot 2018-03-01 at 7 51 57 am

### Other changes:
- Core team section was updated to include the 3 new members
- Many minor screenshots have been updated to include the quickly add `Shapes` shortcut.

Example:
screen shot 2018-03-01 at 8 13 08 am
drill-down-4

### CODE REVIEW

**Comment by junkiattan** [comment]
@damithc @initialshl @Skaty @IanTeo 

We have updated the website with the list of changes. Would be awaiting your reviews and approval before the final deployment

File: _docs/paste-lab.md [hunk 1]
```diff
@@ -25,6 +25,26 @@ The objects are now pasted to fill the entire slide.
 
 Multiple objects can be selected to be used for Paste To Fill Slide, and the pasted objects will automatically be grouped together.
 
+## <a class="anchor-bookmark" id="paste-to-fit-slide"></a> Paste To Fit Slide
+
+**Paste To Fit Slide** helps you paste your copied objects to fit the entire slide.
```
Comment (damithc) [inline _docs/paste-lab.md]: Can this be rephrased to clarify how it differs from `Paste to Fill Slide`. The current description sounds like it fills the entire slide, which is not the case. `Fit` here means maximize size without going out of the slide whereas `Fill` means minimum size that covers the entire slide.

File: _data/coremembers.yml [hunk 2]
```diff
@@ -47,4 +47,22 @@
...
+  name: "Tan Jun Kiat",
+  url: "https://github.com/junkiattan",
+  img: "/img/team/junkiattan.png"
+}
+
+- {
+  name: "Lee Yan Hwa",
+  url: "https://github.com/leeyh20",
+  img: "/img/team/yanhwa.jpg"
+}
```
Comment (damithc) [inline _data/coremembers.yml]: You can move these guys to the 'Past members' section: Min Oo, Hanyang, Yu Ting, Victor, Shunhao

**Review by damithc — CHANGES_REQUESTED** [review]
Some suggestions. Also, consider blurring/cropping out unrelated parts of screenshots (but can be done in a future PR too).

**Comment by damithc** [comment]
On an unrelated note, these should be `Add` `Fix` etc. based on our convention that commit message subject should be in imperative mood :-) 
image

File: _docs/paste-lab.md (hunk 1 above)
Comment (junkiattan) [reply _docs/paste-lab.md]: Hm how about 

>Paste To Fit Slide helps you maximize the size of your copied objects, while staying within the confines of the slide.

Is this acceptable?

Comment (damithc) [reply _docs/paste-lab.md]: Something like this? You can rephrase further as you see fit. 
>_Paste To Fit Slide_ pastes the copied object after resizing the object to the maximum size that can fit inside the slide.

Comment (junkiattan) [reply _docs/paste-lab.md]: How about this?

>Paste to Fit Slide helps to resize the object to the maximum size that can fit inside the slide, before pasting the copied object.

Comment (junkiattan) [reply _docs/paste-lab.md]: Updated!

**Review by damithc — APPROVED** [review]

**Review by initialshl — CHANGES_REQUESTED** [review]
Some of the images (drill down 4, 6) do not have white background erased, can it be done?
Some images look a bit different (possibly different ppt versions), have you tested to see if the resolutions are ok?

**Comment by alexfjw** [comment]
The reduced resolutions are fine as we display them in a smaller res (see the markdown as well).

**Comment by junkiattan** [comment]
@initialshl 
I have erased the background for the 2 images mentioned

**Comment by alexfjw** [comment]
> I have erased the background for the 2 images mentioned

@initialshl cancel that, think there's a few we missed out.

**Comment by alexfjw** [comment]
Minor update, we've resized the 2nd screenshot for the ShapesLab docs and added a box shadow to the first one so styles are consistent in the page.

New version:
new_sc

**Comment by junkiattan** [comment]
@initialshl 

We have changed the images to add in transparent backgrounds.

**Review by initialshl — APPROVED** [review]

**Pull request merged**